src/jit/gentree.h

```diff
@@ -1332,7 +1332,7 @@ struct GenTree
 
     bool OperIsMultiRegOp() const
     {
-#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
+#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_) && defined(ARM_SOFTFP)
```

Comment:
we can just replace `defined(_TARGET_ARM_)` to `defined(ARM_SOFTFP)`.

Reply:
@CarolEidt When I first introduced MultiRegOp, PUTARG_REG and COPY could be MultiRegOp only when ARM_SOFTFP. We can leave these node MultiRegOp when !ARM_SOFTFP, since MultiRegOp could have only one reg. I guess `GetRegCount()` gives actual number of regs used(range is 0-2), so it may be also used where `OperIsMultiRegOp()` is called.

@hseok-oh Regardless of what I said above, MUL_LONG must be always MultiRegOp, while PUTARG_REG and COPY could be depend on arm/armel.

Reply:
@wateret Thanks for your explanation. This fix for `OperIsMultiRegOp()` will be useless if @CarolEidt 's PR is merged